metricq_tools/summary.py

```diff
@@ -0,0 +1,294 @@
+                table[3].append(np.median(self.values[metric]))
+                table[4].append(np.std(self.values[metric]))
+                table[5].append(np.mean(self.values[metric]))
+                table[6].append(np.var(self.values[metric]))
+                table[7].append(len(self.values[metric]))
+            else:
+                table[0].append("n/a")
+                table[1].append("n/a")
+                table[2].append("n/a")
+                table[3].append("n/a")
+                table[4].append("n/a")
+                table[5].append("n/a")
+                table[6].append("n/a")
+                table[7].append(len(self.values[metric]))
+
+        click.echo(tabulate(table, headers=self._metrics, tablefmt="fancy_grid"))
+
+        click.echo()
+        click.echo()
+
+
+async def run_cmd(command):
+    click.echo(f"running... {command!r}")
+
+    proc = await asyncio.create_subprocess_shell(
+        command, stdout=asyncio.subprocess.PIPE, stderr=asyncio.subprocess.PIPE
+    )
+
+    stdout, stderr = await proc.communicate()
+
+    if stdout:
+        click.echo(stdout.decode())
+    if stderr:
+        click.echo(click.style(stderr.decode(), fg="red"))
+    click.echo(f"{command!r} exited with {proc.returncode}")
+
+
+@click.command()
+@metricq_server_option()
+@metricq_token_option(default="metricq-inspect")
+@click.option(
+    "--intervals-histogram/--no-intervals-histogram",
+    "-i/-I",
+    default=False,
+    help="Show an histogram of the observed distribution of durations between data points.",
+)
+@click.option(
+    "--values-histogram/--no-values-histogram",
+    "-h/-H",
+    default=False,
+    help="Show an histogram of the observed metric values.",
+)
+@click.option("--print-data-points/--no-print-data-points", "-d/-D", default=False)
+@click.option("--print-statistics/--no-print-statistics", "-s/-S", default=True)
+@click.option("-m", "--metric", required=True, multiple=True)
+@click_log.simple_verbosity_option(logger, default="WARNING")
+@click.version_option(version=client_version)
+@click.argument("command", nargs=-1)
+def main(
+    server,
+    token,
+    metric,
+    intervals_histogram,
+    values_histogram,
+    print_data_points,
+    print_statistics,
+    command,
+):
+    """Live metric data analysis and inspection on the MetricQ network.
+
+    Consumes new data points for the given metric as they are submitted to the
+    network, prints a statistical overview on exit.
+    """
+
+    asyncio.run(asyc_main(    
+        server,
+        token,
+        metric,
+        intervals_histogram,
+        values_histogram,
+        print_data_points,
+        print_statistics,
+        command
+    ))
+
+
+async def async_main(
+    server,
+    token,
+    metric,
+    intervals_histogram,
+    values_histogram,
+    print_data_points,
+    print_statistics,
+    command,
+):
+    command_str = " ".join(command)
+    summary = Summary(
```

Comment:
```suggestion
    summary = Summary(
        metrics=metric,
```

The `Summary` is missing its list of metrics.